src/File.php

```diff
@@ -727,12 +727,7 @@ protected function onBeforeWrite()
 
         // Update title
         if (!$title) {
-            // Generate a readable title, dashes and underscores replaced by whitespace,
-            // and any file extensions removed.
-            $this->setField(
-                'Title',
-                str_replace(['-','_'], ' ', preg_replace('/\.[^.]+$/', '', $name ?? '') ?? '')
-            );
+            $this->setField('Title', self::normaliseFileName($name));
```

Comment:
```suggestion
            $this->setField('Title', File::normaliseFileName($name));
```
We don't use `self` - see https://docs.silverstripe.org/en/5/contributing/coding_conventions/php_coding_conventions/#other-conventions

Reply:
LOL I was typing below message before seeing this :) All done.